### ISSUE TITLE
qm.te:3:ERROR 'unknown class user_namespace' at token ';' on line 4689:

### ISSUE THREAD

**Issue by dougsland** [containers/qm#84]
CI is failing in this one:
````
Finish: build setup for qm-0.4.2-1.20230611184105946704.pr83.25.gc293cec.el9.src.rpm
Start: rpmbuild qm-0.4.2-1.20230611184105946704.pr83.25.gc293cec.el9.src.rpm
error: %changelog entries must start with *
Building target platforms: x86_64
Building for target x86_64
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.n3K76Z
+ umask 022
+ cd /builddir/build/BUILD
+ cd /builddir/build/BUILD
+ rm -rf qm-0.4.2
+ /usr/bin/gzip -dc /builddir/build/SOURCES/qm-0.4.2.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd qm-0.4.2
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ /usr/bin/git init -q
+ /usr/bin/git config user.name rpm-build
+ /usr/bin/git config user.email '<rpm-build>'
+ /usr/bin/git config gc.auto 0
+ /usr/bin/git add --force .
+ /usr/bin/git commit -q --allow-empty -a --author 'rpm-build <rpm-build>' -m 'qm-0.4.2 base'
+ sed -i 's/^install: man all/install:/' Makefile
+ RPM_EC=0
++ jobs -p
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.lhwK3y
+ umask 022
+ cd /builddir/build/BUILD
+ cd qm-0.4.2
+ /usr/bin/make all
mkdir -p tmp; cp qm.* tmp/
sed -i /"user_namespace exists"/d tmp/qm.if
make -C tmp -f /usr/share/selinux/devel/Makefile qm.pp
make[1]: Entering directory '/builddir/build/BUILD/qm-0.4.2/tmp'
Compiling targeted qm module
qm.te:3:ERROR 'unknown class user_namespace' at token ';' on line 4689:
	allow qm_t self:user_namespace all_user_namespace_perms;
#line 3
/usr/bin/checkmodule:  error(s) encountered while parsing configuration
make[1]: *** [/usr/share/selinux/devel/include/Makefile:157: tmp/qm.mod] Error 1
make[1]: Leaving directory '/builddir/build/BUILD/qm-0.4.2/tmp'
make: *** [Makefile:32: qm.pp] Error 2


RPM build errors:
error: Bad exit status from /var/tmp/rpm-tmp.lhwK3y (%build)
    %changelog entries must start with *
    Bad exit status from /var/tmp/rpm-tmp.lhwK3y (%build)
Finish: rpmbuild qm-0.4.2-1.20230611184105946704.pr83.25.gc293cec.el9.src.rpm
Finish: build phase for qm-0.4.2-1.20230611184105946704.pr83.25.gc293cec.el9.src.rpm
INFO: chroot_scan: 3 files copied to /var/lib/copr-rpmbuild/results/chroot_scan
INFO: /var/lib/mock/centos-stream-9-x86_64-1686508904.151745/root/var/log/dnf.rpm.log
/var/lib/mock/centos-stream-9-x86_64-1686508904.151745/root/var/log/dnf.librepo.log
/var/lib/mock/centos-stream-9-x86_64-1686508904.151745/root/var/log/dnf.log
ERROR: Exception(/var/lib/copr-rpmbuild/results/qm-0.4.2-1.20230611184105946704.pr83.25.gc293cec.el9.src.rpm) Config(centos-stream-9-x86_64) 0 minutes 25 seconds
INFO: Results and/or logs in: /var/lib/copr-rpmbuild/results
INFO: Cleaning up build root ('cleanup_on_failure=True')
Start: clean chroot
INFO: unmounting tmpfs.
Finish: clean chroot
ERROR: Command failed: 
 # /usr/bin/systemd-nspawn -q -M c276c3673249462582fd144d2a2de0be -D /var/lib/mock/centos-stream-9-x86_64-1686508904.151745/root -a -u mockbuild --capability=cap_ipc_lock --rlimit=RLIMIT_NOFILE=10240 --capability=cap_ipc_lock --bind=/tmp/mock-resolv.zbcp2kly:/etc/resolv.conf --bind=/dev/btrfs-control --bind=/dev/mapper/control --bind=/dev/loop-control --bind=/dev/loop0 --bind=/dev/loop1 --bind=/dev/loop2 --bind=/dev/loop3 --bind=/dev/loop4 --bind=/dev/loop5 --bind=/dev/loop6 --bind=/dev/loop7 --bind=/dev/loop8 --bind=/dev/loop9 --bind=/dev/loop10 --bind=/dev/loop11 --console=pipe --setenv=TERM=vt100 --setenv=SHELL=/bin/bash --setenv=HOME=/builddir --setenv=HOSTNAME=mock --setenv=PATH=/usr/bin:/bin:/usr/sbin:/sbin --setenv=PROMPT_COMMAND=printf "\033]0;<mock-chroot>\007" --setenv=PS1=<mock-chroot> \s-\v\$  --setenv=LANG=C.UTF-8 --resolv-conf=off bash --login -c /usr/bin/rpmbuild -bb  --target x86_64 --nodeps /builddir/build/SPECS/qm.spec

Copr build error: Build failed
```

**Comment by dougsland** [comment]
@rhatdan I was wondering if this was fixed via: https://github.com/containers/qm/commit/013687d21dee7c3165e9d4b580bac9ae72e28ab9 and somehow not flying in the CI ?

**Comment by dougsland** [comment]
@Yarboa any ideas here?

**Comment by dougsland** [comment]
resolved by #94
